zonos/backbone/_mamba_ssm.py

```
@@ -5,11 +5,38 @@
+    attention layers (for quality) based on the configuration.
+    
+    Mamba SSMs provide linear scaling with sequence length while maintaining
+    strong modeling capabilities, making them ideal for long audio sequences.
+    
+    Attributes:
+        supported_architectures (list): Architecture types this backbone supports
+        config (BackboneConfig): Configuration containing layer specifications
+        layers (nn.ModuleList): Stack of Mamba/attention blocks  
+        norm_f (nn.LayerNorm): Final layer normalization
+    """
     supported_architectures = ["transformer", "hybrid"]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)